101-acs-dcos/README.md

```diff
@@ -0,0 +1,17 @@
+# Azure Container Service
```

Comment:
@rgardler There's a 101-acs-mesos template as well. Does that need to be removed?